### ISSUE TITLE
Indent size setting is not saved beyond page session.

### ISSUE THREAD

**Issue by nmeyvis** [jsfiddle/jsfiddle-issues#1452]
Issue: Indent size is not saved beyond page session.

Expected: When I change the Indent size from 2 to 4, I expect that setting to be the same when I open a new fiddle page, especially when I'm signed in.

Reproduce: Open fiddle, change indent size, open another fiddle and see the setting is reverted.

It's a bit frustrating that every time I use jsfiddle I have to mess with the settings.

**Comment by nmeyvis** [comment]
It also seems like the only setting that is saved is the Dark Theme.

**Comment by AlexeyNoir** [comment]
I got the same issue. When I type a valid code, it autoruns per every key press. In the "behavior" section of the settings I disabled "Only auto-run code that validates", reloaded the page and it still remains enabled. Pisses me off. 
Btw, Dark theme parameter is also becomes disabled after F5, but still works. The "Editor layout" is the only param that is OK.
![0](https://user-images.githubusercontent.com/43852391/57364334-a094a600-718b-11e9-9a58-f21f863bc889.jpg)
